### ISSUE TITLE
Publish FE Tools documentation

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/va.gov-team#29436]
## Narrative
Once all of the documentation has been migrated to the [TEMP - FE Tools Documentation Migration](https://vfs.atlassian.net/wiki/spaces/FTT/pages/1828520012/TEMP+-+FE+Tools+Documentation+Migration) section in Confluence and it has been reviewed, it will be ready to be published on the [new documentation site](https://depo-platform-documentation.scrollhelp.site/index.html)!

We will need to migrate all of the documents inside of the TEMP section to the [Frontend developer documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/Frontend-developer-documentation.687931428.html) section of the new documentation site.

Info about this process can be found in the [Content migration guidelines](https://vfs.atlassian.net/wiki/spaces/AP/pages/1375666200/Content+migration+guidelines)

## In Scope
Moving documents in the current structure from the TEMP section in the FE Tools Confluence space to the Frontend developer documentation section of the new documentation space.

## Acceptance Criteria
- [ ] All documents from the TEMP section have been moved to the new documentation Confluence space.
- [ ] Documentation structure from the TEMP section has been maintained.
- [ ] Alert Content & IA team in the vsp-confluence channel

**Comment by k80bowman** [comment]
Since @ndsprinkle is currently working on updating several of our deployment docs with information about Github Actions, and since we are not yet using Github Actions for deployments, I'm going to publish everything except the Deployments section for now. We can publish the Deployments documentation (and deprecate the current documentation) once it is ready to go.

**Comment by k80bowman** [comment]
Everything but the deployment section is now live on the scroll viewport site! https://depo-platform-documentation.scrollhelp.site/developer-docs/Frontend-developer-documentation.687931428.html

I've asked the team to take a quick tour and let me know if they find anything weird. If nothing comes up, I plan on closing this card Monday.

**Comment by k80bowman** [comment]
Suggested Changes:
1. From [this thread](https://dsva.slack.com/archives/C01AS05BQ79/p1630685685039600?thread_ts=1630684198.039200&cid=C01AS05BQ79): For Step 1: Set up Node on Linux, it would be much better to recommend installing nvm via git rather than the curl | bash approach.
I’ve seen use of it in the industry but it’s a security issue that should be avoided.

**Comment by k80bowman** [comment]
The docs are live and the fix recommended above is complete! Closing this one for now. I will create a new ticket to publish and then deprecate the deployment docs once @ndsprinkle is done updating them.
